File: 백준/Bronze/10817. 세 수/세 수.java
import java.util.Scanner;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int[] scores = new int[input.length];
        
        for (int i = 0; i < input.length; i++) {
            scores[i] = Integer.parseInt(input[i]);
        }
        
        Arrays.sort(scores);
        System.out.println(scores[1]);
        
        scanner.close();
    }
}

File: 백준/Bronze/10817. 세 수/세 수.cs
namespace ConsoleApp1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split();
            int[] scores = Array.ConvertAll(input, int.Parse);
            
            Array.Sort(scores);

            Console.WriteLine(scores[1]);
        }
    }
}
